pkg/apps/strategy/recreate/recreate_test.go

```diff
@@ -98,14 +99,14 @@ func (c *fakeControllerClient) fakeScaleClient() *scalefake.FakeScaleClient {
 		updateAction := action.(clientgotesting.UpdateAction)
 		scaleObj := updateAction.GetObject().(*autoscalingv1.Scale)
 		c.scaleEvents = append(c.scaleEvents, scaleObj)
-		c.deployment.Spec.Replicas = scaleObj.Spec.Replicas
+		c.deployment.Spec.Replicas = &scaleObj.Spec.Replicas
```

Comment:
I'd consider shallow copy a bad pattern in general, probably no side effect here

Reply:
i can deep copy but for the test it probably does not matter